packages/lib/src/index.ts

```diff
@@ -3,6 +3,7 @@ export { default as GalleryItem } from './core/galleryItem';
 export { default as BlueprintsManager } from './core/blueprints/BlueprintsManager';
 export { default as blueprints } from './core/blueprints/Blueprints';
 export { default as defaultStyles } from './common/defaultStyles';
+export { default as duplicatedInWix } from './common/duplicatedInWix';
```

Comment:
Maybe should be renamed to "duplicatedStylesInWix" or something luke that, cause when imported from "pro-gallery" it will not be cleated what is duplicated.

Reply:
We also try to so not use the word "wix" in the core, not have a better idea for a naming but it's something to think about.

Reply:
I can call it coreStyles instead of using wix.